src/flows/pipes/Schedule/ExportTaskButton.tsx

```diff
@@ -97,58 +106,68 @@ const ExportTaskButton: FC<Props> = ({
       }
     }
 
-    postTask({data: {orgID: org.id, flux: query}})
-      .then(resp => {
-        if (resp.status !== 201) {
-          throw new Error(resp.data.message)
-        }
+    const resp = await postTask({data: {orgID: org.id, flux: query}})
+    try {
```

Comment:
nit: just in case the org is undefined for some reason
```suggestion
    try {
      const resp = await postTask({data: {orgID: org.id, flux: query}})
```

Reply:
good catch. done.